src/panel/common/commonActions.js

```diff
@@ -66,9 +70,11 @@ function isInvalidTitle(title) {
 	}
 }
 
-function isInvalidTags(tagArr) {
-	for (let i=0; i<tagArr.length; i++) {
-		if (tagArr[i] === "") {
+
+function isInvalidTags(tokenArr) {
+	for (let i=1; i<tokenArr.length; i++) {
+		// Tags are invalid if there are two consecutive operators
+		if (tokenArr[i-1]["tokenType"] === "BINARY_OP" && tokenArr[i]["tokenType"] === "BINARY_OP") {
```

Comment:
Yep... Had forgotten that the token array was in rpn,  thought I was iterating through an infix array. I'll implement your idea :)